Header header
string motorId
Position[] positions
